package.json

```diff
@@ -43,5 +47,9 @@
     "ts-node": "~8.3.0",
     "tslint": "~6.1.0",
     "typescript": "~4.0.2"
+  },
+  "engine": {
```

Comment:
Hopefully per the stack overflow this works. Great work! Thanks for driving through this.